docs/source/dev_guide/algorithms/pca/computation-batch.rst

```diff
@@ -0,0 +1,175 @@
+   * - correlation
+     - Use when the input data is a correlation matrix. Pointer to the :math:`p \times p`
+       numeric table that contains the correlation matrix. This input can be an
+       object of any class derived from NumericTable except
+       PackedTriangularMatrix.
+
+Algorithm Parameters
+--------------------
+
+The PCA algorithm has the following parameters, depending on the
+computation method parameter method:
+
+.. list-table::
+   :widths: 10 10 15 25
+   :header-rows: 1
+   :align: left
+
+   * - Parameter
+     - method
+     - Default Value
+     - Description
+   * - algorithmFPType
+     - defaultDense or svdDense
+     - float
+     - The floating-point type that the algorithm uses for intermediate
+       computations. Can be float or double.
+   * - method
+     - Not applicable
+     - defaultDense
+     - Available methods for PCA computation:
+
+       For CPU:
+
+       - ``defaultDense`` - the correlation method
+       - ``svdDense`` - the SVD method
+
+       For GPU: 
+
+       - ``defaultDense`` - the correlation method
+
+
+   * - covariance
+     - defaultDense
+     - SharedPtr<covariance::Batch<algorithmFPType, covariance::defaultDense> >
+     - The correlation and variance-covariance matrices algorithm to be used
+       for PCA computations with the correlation method.
+   * - normalization
+     - svdDense
+     - SharedPtr<normalization::zscore::Batch<algorithmFPType,
+       normalization::zscore::defaultDense>>
+     - The data normalization algorithm to be used for PCA computations with
+       the SVD method. 
+   * - nComponents
+     - defaultDense, svdDense
+     - 0
+     - Number of principal components :math:`p_r`. If it is zero, the algorithm
+       will compute the result for :math:`p_r = p`.
+   * - isDeterministic
+     - defaultDense, svdDense
+     - false
+     - If true, the algorithm applies the "sign flip" technique to the results.
+   * - resultsToCompute
+     - defaultDense, svdDense
+     - none
+     - The 64-bit integer flag that specifies which optional result to compute.
+
+       Provide one of the following values to request a single characteristic
+       or use bitwise OR to request a combination of the characteristics:
+
+       -  mean
+       -  variance
+       -  eigenvalue
+
+Algorithm Output
+----------------
+
+The PCA algorithm calculates the results described below. Pass the
+Result ID as a parameter to the methods that access the results of
+your algorithm. 
+
+.. list-table::
+   :widths: 25 25
+   :header-rows: 1
+   :align: left
+
+   * - Result ID
+     - Result
+   * - eigenvalues
+     - Pointer to the :math:`1 \times p_r` numeric table that contains eigenvalues
+       in the descending order. By default, this result is an object of the
+       HomogenNumericTable class, but you can define the result as an object of
+       any class derived from NumericTable except PackedSymmetricMatrix,
+       PackedTriangularMatrix, and CSRNumericTable.
```

Comment:
In ALS algorithm the text "This input can be an object of any class ..." is formatted as a Note statement. Should it be formatted the same way across all algorithms?
The same for 'eigenvectors'.

Reply:
Yes, you are right. But at the moment there are still instances where this text isn't converted into a note, this is not the only one. I'm changing this gradually.